truefoundry.yaml

```diff
@@ -11,7 +11,6 @@ components:
       MODELS_CONFIG_PATH: ./models_config.truefoundry.yaml
       METADATA_STORE_CONFIG: '{"provider":"prisma"}'
       ML_REPO_NAME: cognita-internal
-      INFINITY_URL: http://cas-infinity.cognita-internal.svc.cluster.local:8000
```

Comment:
Why is this being removed?

Reply:
the url is present in the config.yaml file hence.